### DISCUSSION TITLE
Ideas for names

### DISCUSSION THREAD

**Discussion by edmondchuc** [Kurrawong/vocedit#3] [Q&A] [answered]
Please add a new comment for a new name idea. Otherwise upvote the name that you like best!

**Comment by edmondchuc** [comment]
From Nick:

> VocSimple - "it's just VocSimple..."

**Comment by edmondchuc** [comment]
VocForm

**Comment by edmondchuc** [comment]
VocSmith

**Reply by Metaduck** [reply]
Naturally

**Comment by edmondchuc** [comment]
From Sophie:

> VocEase

**Reply by leskneebone** [reply]
Liked @SLDarnell other suggestion. This one sounds too medical. Like something that helps you get a vocab through your system (actually now that I've written that... not so bad).

**Comment by edmondchuc** [comment]
From David:

> VocVoyage

**Comment by edmondchuc** [comment]
From Howard:

> VocLite

**Comment by edmondchuc** [comment]
From Howard:

> VocStudio

**Comment by edmondchuc** [comment]
From Sophie:

> VocGo

**Comment by edmondchuc** [comment]
From Les:

> VocBox

**Reply by leskneebone** [reply]
Don't vote for this. it sounds like a DJ mixer.

**Comment by edmondchuc** [comment]
From Les:

> VocZen

**Comment by edmondchuc** [comment]
From Margie:

> VocPlan

**Comment by edmondchuc** [comment]
From Lawson:

> VocEdit

**Reply by hjohns** [reply]
We could use an image from VocEdit.com...
![image](https://github.com/user-attachments/assets/72ce465d-922f-4a32-87c6-2ac6f93e42b8)

**Reply by lalewis1** [reply]
good god. really makes you want to buy there software 🤣

**Comment by edmondchuc** [comment]
From Sophie:

> VoKit

**Reply by leskneebone** [reply]
Love this because it can be used as a noun, a verb, and an exclamation:

- vocabularies kit: - a set of tools
- vok it: to make something a vocabulary
- vokit! : as in, "vokit - let's make some triples, for vok sake"

**Reply by leskneebone** [reply]
"Trouble formalising that pesky nomenclature or framework? Just Vokit!"

**Comment by edmondchuc** [comment]
From Margie:

> VocPolish

**Comment by edmondchuc** [comment]
From David:

> VocCelerator

**Comment by hjohns** [comment, marked as answer]
VocEditor
<img src="https://github.com/user-attachments/assets/a7ea89bc-d210-4bc8-8cde-7ab1503ba1f3" width="400">

**Reply by nicholascar** [reply]
I'll vote for this if it's shortened to just "VocEdit"!

**Reply by recalcitrantsupplant** [reply]
Sounds good to me - the image reminds me of a book I have ...
![out](https://github.com/user-attachments/assets/c72caa02-2bec-4e7b-a345-96e33240a538)
